map.go

```diff
@@ -387,6 +418,21 @@ func (m *EbpfMap) isPerCpu() bool {
 		m.Type == MapTypePerCpuCGroupStorage
 }
 
+// Per-CPU maps require extra space to store values from ALL possible CPUs.
+// For access from userspace, each single value is padded so that it's a multiple of 8 bytes.
+func (m *EbpfMap) setValueRealSize() error {
+	if m.isPerCpu() {
+		numCpus, err := GetNumOfPossibleCpus()
+		if err != nil {
+			return err
+		}
+		m.valueRealSize = ((m.ValueSize + 7) / 8) * 8 * numCpus
```

Comment:
Could you please add comment where these values come from?

Reply:
Absolutely, I guess just a link to the kernel commit I linked in the pr description?

Reply:
Yeah, PR description is pretty comprehensive.. :)